sorting/pancake_sort.cpp

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright 2020 @author Abhishek Ajoy Menon
+ *
+ * @file
+ *
+ * A generic implementation of pancake_sort.
+ */
+#include<iostream>
+/* Sorting an array of integers in ascending order
+
+ * The algorithm finds the largest number in the array
+ * and moves it to the end of the array.
+ * This moving of array to the end is done by reversing the array
+ * from the start to the location of the largest number (which 
+ * brings the largest number to the beginning) and then reversing 
+ * the whole array. The reversing process is analogous to flipping
+ * pancakes, hence the name.
+*/
+/**
+ *  flip(): Reverses arr[0..i]
+ */
+void flip(int arr[], int i) {
+    int temp, start = 0;
+    while (start < i) {
+        temp = arr[start];
+        arr[start] = arr[i];
+        arr[i] = temp;
+        start++;
+        i--;
+    }
+}
+/**
+ * findMax(): Returns index of the  maximum element in arr[0..n-1]
+ */
+int findMax(int arr[], int n) {
```

Comment:
see changes suggested above for documentation of function and function arguments